decidim_app-design/app/views/public/partials/_focus_content_content_1.html.erb

```diff
@@ -1,26 +1,26 @@
 <h1 class="heading2">
-  ¿Estás de acuerdo con que se imponga una cuota de 40€ al año por miembro?
+  ¿Con cuáles de estas afirmaciones sobre la organización del Comité de Fiestas estás&nbsp;de&nbsp;acuerdo?
```

Comment:
Just a little thing, that maybe it's not important enough, but as you changed the language translation text in a design template above:
- decidim_app-design/app/views/public/evote-poll-landing.html.erb

![image](https://user-images.githubusercontent.com/9702463/90139845-2c2ab880-dd79-11ea-8725-c19b26caa0e0.png)

it would be better maintain same design language in all place ? So, or maintain spanish language above or change to english language here too.
I think the same thing happens in next design templates..

Reply:
Hi @Leusev,
thanks a lot for the review. I didn't change any translations "just" merged the new version from the [evote-design-branch](https://github.com/decidim/decidim/tree/feat/evote) into my branch. Sorry if that was confusing. 🙏

Reply:
Sorry for that doubled comment's mess @slickepinne ... My computer had freezed & had to restart while doing pr review and I've realised that github had saved some comments..

Reply:
Haha, don't worry at all. I prefer having too many comments instead of having not enough. 😊